protostar/starknet/cheater.py

```diff
@@ -3,6 +3,12 @@
 from typing_extensions import Self
 
 
+class CheaterException(Exception):
+    def __init__(self, message: str):
+        super().__init__(message)
+        self.message = message
```

Comment:
There is no need here for the `__init__` method. If you inherit from `Exception` you can do `str(CheaterException("message"))` and it will yield `"message"`